index.js

```diff
@@ -63,3 +63,8 @@ function getStat(config, stat) {
   clientTransactionId++;
   return response;
 };
+
+// We're going to round our numbers up (in the positive direction) to the nearest <precision>th place
+function roundToPrecision(num, precision) {
+  return +Number.parseFloat(num).toPrecision(precision);
+};
```

Comment:
Nit picky (doesn't really matter), but technically a function statement shouldn't have a semicolon at the end.

Reply:
Yeah, I can change that before merging if you'd like.